src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Altinn.Platform.Authentication.Model;
+using Altinn.Platform.Authentication.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Platform.Authentication.Controllers
+{
+    /// <summary>
```

Comment:
I  think it's good to have a feature flag for system user so we can deploy it in the AT environments in the coming weeks for testing, So we have the possibility to turn off the systemuser feature in TT02/prod but have it on in AT environments